src/networking.c

```diff
@@ -543,10 +543,14 @@ void afterErrorReply(client *c, const char *s, size_t len, int flags) {
     if (!(flags & ERR_REPLY_FLAG_NO_STATS_UPDATE)) {
         /* Increment the global error counter */
         server.stat_total_error_replies++;
-        /* Increment the error stats
-         * If the string already starts with "-..." then the error prefix
-         * is provided by the caller ( we limit the search to 32 chars). Otherwise we use "-ERR". */
-        if (s[0] != '-') {
+        /* Increment the error stats */
+        /* After the errors RAX reaches this limit, instead of tracking
+         * custom LUA errors, we track the error under `error_LUA` */
+        if (flags & ERR_REPLY_FLAG_LUA && raxSize(server.errors) >= ERROR_STATS_LUA_LIMIT) {
+            incrementErrorCount(LUA_ERRORSTATS_OVERFLOW_ERR, strlen(LUA_ERRORSTATS_OVERFLOW_ERR));
```

Comment:
If the error is already in the rax, I think we should increment it even if the rax size is over the limit. Incementing an existing counter does't add a new element to the rax.

Reply:
Yes, we can do a check to see if the error exists already in the RAX and allow increments. I will make this change

Reply:
Updated here: https://github.com/valkey-io/valkey/pull/500/commits/fe6e57423dcfdee5853dfc785d25191f485e484a